index.js

```diff
@@ -1,4 +1,6 @@
 require('dotenv').config()
+require('https').globalAgent.options.rejectUnauthorized = false;
```

Comment:
Is there a reason you are changing the TLS behaviour here, @frenkel34 using weird certs again?

Reply:
Yes its the only way I could get it to work. Removed it from repository, will use locally if needed.